clients/admin-ui/src/features/auth/ProtectedRoute.tsx

```diff
@@ -29,23 +30,19 @@ const useProtectedRoute = (redirectUrl: string) => {
 };
 
 interface ProtectedRouteProps {
-  redirectUrl: string;
-  authenticatedBlock: JSX.Element;
-  children: JSX.Element;
+  children: ReactNode;
```

Comment:
`ReactNode` lets this wrap any React content - it just has to handle chakra's annoying JSX complaint with the fragment comment below.